php/create-team.php

```diff
@@ -23,7 +38,22 @@
 		if (!$stmt->execute())
 			echo 'false';
 		else
+		{
+			$stmt->close ();
+
+			$stmt= $db->prepare('SELECT LAST_INSERT_ID()');
+	        $stmt->execute();
+			$stmt->bind_result($team_id);
+			$stmt->fetch();
+			$stmt->close();
+
+			$stmt = $db->prepare('INSERT INTO collaborate VALUES (?, ?)');
+			$stmt->bind_param('ii', $manager_id, $team_id);
+			$stmt->execute();
+			$stmt->bind_result($result);
```

Comment:
you can't bind result nor fetch to statements which do operations like insert and update (i.e., that ones which don't return a result).